storage/innobase/include/ut0counter.h

```diff
@@ -39,7 +39,7 @@ Created 2012/04/12 by Sunny Bains
 #endif /* CPU_LEVEL1_DCACHE_LINESIZE */
 
 /** Default number of slots to use in ib_counter_t */
-#define IB_N_SLOTS		64
+#define IB_N_SLOTS		128
```

Comment:
Do we really need this `#define`? Generally, I would prefer `constexpr`, but it seems to me that this definition is unnecessary. We could simply write:
```c++
template <typename Type, int N>
struct ib_counter_t {
```
The only instantiation of `ib_counter_t` in `srv_stats_t` is explicitly specifying the parameter `N`.

Reply:
done.